### DISCUSSION TITLE
StateContainer property for animating state changes

### DISCUSSION THREAD

**Discussion by Cybrosys** [CommunityToolkit/Maui#1580] [New Feature Discussions]
Hello,

Would it be possible to add a property or similar so the state container animates the state changes?

I am not a fan of calling the static method `ChangeStateWithAnimation` as it breaks the ability to data-bind the current state between my xaml and view model.

For example:
`toolkit:StateContainer.Animate="true"`

**Comment by CliffAgius** [comment]
Closed as per the Community Stand-up discussion here: https://youtu.be/N9wMcBP4jtg?t=2889 where it was agreed that New Features need to get above 7 upvotes within 6 months to progress to being reviewed by the maintainers.
